.github/workflows/post-release-changes.yaml

```diff
@@ -0,0 +1,51 @@
+name: Post-release changes
+
+on:
+  release:
+    types: [published]
+
+env:
+  PYTHON_VERSION: 3.9
+
+jobs:
+  post-release-changes:
+    runs-on: ubuntu-latest
+    steps:
+    # setup pre-reqs
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0 # fetch all commits/branches
+    - name: Setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ env.PYTHON_VERSION }}      
+    - name: Get version
+      run: |
+        tarref=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+        if [[ "${{ github.ref }}" == "refs/tags/"* ]]; then
+          # Strip "v" prefix from tagref
+          echo "VERSION=$(echo $tarref | sed -e 's/^v//')" >> $GITHUB_ENV
+          echo "MAJOR_MINOR_VERSION=$(echo $tarref | sed -e 's/^v//' -e 's,\([0-9]*\.[0-9]*\)\.\([0-9]*\),\1,')" >> $GITHUB_ENV
+        elif [[ "${{ github.ref }}" == *"master" ]]; then
+          echo "VERSION=latest" >> $GITHUB_ENV
+        fi
+    - name: Change install version
+      run: |
+        pip install -r ./.github/workflows/requirements.txt
+        python3 ./.github/workflows/change-install-version.py ${{ env.VERSION }}
+        # Work around: YAML parser does not correctly dump tags ("!%21" instead of "!!")
```

Comment:
Note this extra step. I struggled with this issue and I have concluded that instead of trying to understand why the parser behaves the way it does, this workaround is a better use of our time.